### PR TITLE
WT-9797 Add more trace messages to test format.

### PR DESCRIPTION
**Pull request by sueloverso**

### CODE REVIEW

File: test/format/backup.c [hunk 1]
```diff
@@ -466,7 +470,7 @@ backup(void *arg)
     WT_CURSOR *backup_cursor;
     WT_DECL_RET;
     WT_SESSION *session;
-    u_int counter, incremental, period;
+    u_int counter, incremental, period, num_yield;
```
Comment (keitharnoldsmith) [inline test/format/backup.c:473]: Alphabetize :-)

File: test/format/backup.c [hunk 2]
```diff
@@ -612,8 +622,11 @@ backup(void *arg)
             testutil_check(session->truncate(session, "log:", backup_cursor, NULL, NULL));
 
         testutil_check(backup_cursor->close(backup_cursor));
-        trace_msg(session, "Backup #%u stop%s%s%s", counter, config == NULL ? "" : ": (",
-          config == NULL ? "" : config, config == NULL ? "" : ")");
+        if (config == NULL)
```
Comment (keitharnoldsmith) [inline test/format/backup.c:625]: Definitely an improvement in readability!

**Review by keitharnoldsmith — APPROVED** [review]
LGTM. Only one very minor comment.

File: test/format/backup.c (hunk 2 above)
Comment (sueloverso) [reply test/format/backup.c:625]: And it matches the (already existing) "start" trace message. And that's why I made the checkpoint one the same as well. More readable and consistent.

**Comment by sueloverso** [comment]
The PR testing failure is a python test getting WT_ROLLBACK. It is unrelated to these changes. I will merge.

**Pull request merged**